charts/jenkins/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Use the following links to reference issues, PRs, and commits prior to v2.6.0.
 The changelog until v1.5.7 was auto-generated based on git commits.
 Those entries include a reference to the git commit to be able to get more details.
 
+## 5.0.19
+
+Add agent.restrictedPssSecurityContext to automatically inject in the jnlp container a securityContext that is suitable for the use of the restricted Pod Security Standard
```

Comment:
```suggestion
Add `agent.restrictedPssSecurityContext` to automatically inject in the jnlp container a securityContext that is suitable for the use of the restricted Pod Security Standard
```

Reply:
Fixed - I saw a mix of one and the other in the existing entries and wasn't sure which direction to go